ci/citest/unit-tests/unit-tests.sh

```diff
@@ -46,5 +46,6 @@ docker cp . "${CID}:/var/tmp/go/src/github.com/axsh/openvdc"
 
 ## Run unit tests
 docker exec $CID /bin/bash -c "/usr/bin/env"
-docker exec $CID /bin/bash -c "cd /var/tmp/go/src/github.com/axsh/openvdc;  govendor sync"
+#docker exec $CID /bin/bash -c "cd /var/tmp/go/src/github.com/axsh/openvdc;  govendor sync"
```

Comment:
I assume this was accidentally committed? :)

Reply:
? I'm confused. That's not what I've got (and it's not what's just run - and failed - on the ci).
Pushing soon.